### ISSUE TITLE
Cost vs impression 

### ISSUE THREAD

**Issue by muuratjann** [google/lightweight_mmm#52]
Hi team,

I have some question regarding the model. I would be very happy if you could have a look at them :)

1. In the theory section of the model you say "media channels is a matrix of different media channel activity: typically impressions or costs per time period". In my data I do not have impressions but I only have costs. So, is it still possible to use optimisation tool? If yes, how do we define "prices – An array with shape (n_media_channels,) for the cost of each media channel unit?" 

2. As you have suggested the r_hats are all less than 1.1. But in the bottom I see that there are 107 divergencies. What does it mean and what is the definition of r_hat?
<img width="1061" alt="Screenshot 2022-08-12 at 22 30 41" src="https://user-images.githubusercontent.com/49808871/184440005-9b8f79d5-242d-4294-ad08-75e4b8bd3ce2.png">

3. I know that the "media_effect_hat" is just defined as the coefficient of the media, and how is the second metric defined mathematically?
<img width="821" alt="Screenshot 2022-08-12 at 22 33 33" src="https://user-images.githubusercontent.com/49808871/184440870-8ff28431-4336-44e6-81ea-6da2d4591db5.png">

**Comment by pabloduque0** [comment]
Hello @muuratjann !

1.  If you are using spend/cost as your media unit for all channels then your prices will always be an array of 1s. Eg if you have 5 channels `jnp.ones(5)` should do the trick.
2. I will refer you to a couple resources where you can read both about [r_hat](https://mc-stan.org/rstan/reference/Rhat.html) and [divergences](https://mc-stan.org/users/documentation/case-studies/divergences_and_bias.html). 
3. The ROI includes the cost into the contribution calculation in order to calculate Return On Investment. It is calculated the follwoing `roi_hat = sum_unscaled_prediction * media_contribution_hat / unscaled_costs`

**Comment by muuratjann** [comment]
Thank you a lot @pabloduque0 !

Another question I have: is there a reason why my optimisation algorithm is running forever (media_features.shape=(1200,9), n_time_periods=10, no extra features, 50 parameters in my model)? What are the possible remedies to decrease that?

**Comment by pabloduque0** [comment]
Our current bounds might be too flexible (which is something we will change soon). So maybe try reducing `bounds_lower_pct` and `bounds_upper_pct` to 0.05. But its just a guess, I need a bit more info on the specific inputs you are feeding to the function to check.

**Comment by pabloduque0** [comment]
Closing due to inactivity, feel free to re-open if there are any questions/issues left.

**Comment by mjjohns1** [comment]
@pabloduque0 Possible to add information about how to set up the data and model when you only have spend data to the documentation and perhaps add an example in a notebook? I suspect that most users will only have cost data, making the the situation used in the current examples (i.e., separate information on impressions and costs) the exception and not the rule.
